.github/risk_level_guide.md

```diff
@@ -0,0 +1,23 @@
+## Risk level guide
+
+**Low risk**: with high confidence the change won’t unexpectedly change the behaviors, or the change touches a non-critical area.
```

Comment:
I suggest that `behavior` (singular) would be better here (and in the lines below).

Reply:
Not a problem. Updated in [cc4672b](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/cc4672b30244322dcdbbebefb7d79467f15639ca)